plugins/module_utils/redfish_utils.py

```diff
@@ -1814,6 +1876,11 @@ def get_fan_inventory(self):
                 return response
             result['ret'] = True
             data = response['data']
+            #To support HPE operations
+            if "Oem" in data.keys():
```

Comment:
This is an OEM extension and does not belong in the standard module. Any OEM extensions should be in their own module.

Reply:
This has been removed in the latest commit.